src/Arcus.Messaging.Tests.Workers.ServiceBus.Topic/Dockerfile

```
@@ -1,7 +1,7 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.0-alpine3.9 AS base
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.202-alpine3.10 AS base
 WORKDIR /app
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.0-alpine3.9 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.202-alpine3.10 AS build
```

I think we shouldn't be too specific about the version but yours is also ok; don't have a pref
```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine3.11 AS build
```